hw/drivers/i2s/src/i2s.c

```diff
@@ -125,10 +126,16 @@ i2s_buffers_from_pool(struct i2s *i2s, struct i2s_buffer_pool *pool)
     sample_data = (uintptr_t)&buffers[pool->buffer_count];
 
     for (i = 0; i < pool->buffer_count; ++i) {
+        for (j = 0; j < I2S_BUFFER_GUARD; ++j, ++sample_data) {
+            *(uint8_t *)sample_data = 0xFD;
```

Comment:
Another nit-pick: Maybe define the buffer guard value and use the definition instead of the hard-coded value.